build/AzurePipelinesTemplates/WindowsAppSDK-BuildTransportPackage-Stage.yml

```diff
@@ -0,0 +1,195 @@
+      # read more about custom job pool types at https://aka.ms/obpipelines/yaml/jobs
+      type: windows
+    timeoutInMinutes: 120
+    dependsOn:
+    - BuildBinaries
+    - BuildBinaries_release_anycpu
+    - BuildAndTestMRT
+    condition: or(succeeded(), eq(${{ parameters.IgnoreFailures }}, 'true'))
+    variables:
+      ob_outputDirectory: '$(REPOROOT)\out'
+    steps:
+    - template: WindowsAppSDK-PackNuget-Steps.yml@self
+      parameters:
+        SignOutput: ${{ parameters.SignOutput }}
+
+  # Build WinAppSDK and Run Integration Test from TestAll.ps1
+  - job: WinAppSDKIntegrationBuildAndTest
+    dependsOn: ['NugetPackage']
+    condition: succeeded()
+    pool:
+      type: windows
+      isCustom: true
+      name: 'ProjectReunionESPool-2022' # This stage is purely a test stage so we will use our own pools here
+    variables:
+      WindowsAppSDKTransportPackageVersion: $[ dependencies.NugetPackage.outputs['SetVersion.packageVersion'] ]
+      VersionWithDevTag: $[format('{0}.{1}.{2}-{3}.{4}.{5}', variables['major'], variables['minor'], variables['patch'], 'dev', variables['versionDate'], variables['versionCounter'])]
+    steps:
+      #TODO: Remove the two template calls once Add-AppxPackage has been resolved
+      - template: ..\..\eng\common\AzurePipelinesTemplates\WindowsAppSDK-BuildSetup-Steps.yml
+        parameters:
+          AzureSubscriptionServiceConnection: 'ProjectReunion Resource Manager'
+          NuGetServiceConnectionName: 'ProjectReunionInternal'
+
+      - template: ..\..\eng\common\AzurePipelinesTemplates\WindowsAppSDK-Build-Steps.yml
+        parameters:
+          TransportPackageArtifactName: 'drop_TransportPackage_NugetPackage'
+          WindowsAppSDKPackageVersion: $(VersionWithDevTag)
+          BuildType:  'stable'
+          TransportPackages:
+            FoundationPackage:
+              Name: Microsoft.WindowsAppSDK.Foundation.TransportPackage
+              Version: $(WindowsAppSDKTransportPackageVersion)
+
+      # TODO: Reenable, Add-AppxPackage currently does not work on 2022 images. See issue #2908
+      # - template: ..\..\eng\common\WindowsAppSDK-BuildAndIntegrationTest-Steps.yml
+      #   parameters:
+      #     WindowsAppSDKPackageVersion: $(VersionWithDevTag)
+      #     TestSelection: '*Foundation_*'
+      #     TransportPackageArtifactName: 'FullNuget'
+      #     BuildType: 'stable'
+      #     TransportPackages:
+      #       FoundationPackage:
+      #         Name: Microsoft.WindowsAppSDK.Foundation.TransportPackage
+      #         Version: $(WindowsAppSDKTransportPackageVersion)
+
+  # Publish
+  - job: Publish
+    dependsOn:
+    - NugetPackage
+    # - WinAppSDKIntegrationBuildAndTest
+    condition: and(or(succeeded(), eq(${{ parameters.IgnoreFailures }}, 'true')), eq('${{ parameters.publishToMaestro }}', 'true'))
+    pool:
+      type: windows
+      isCustom: true
+      name: 'ProjectReunionESPool-2022' # This stage is purely for ES, hence this custom pool
```

Comment:
At the top of this yml, the pool has already been set for this stage. If we want to also explicitly set the pool for this job, wouldn't it be clearer to say: "This job is..."?

Reply:
Correct! Will update